src/client/pages/Geo/GeoMap/components/ButtonStatisticsTableExport/utils.ts

```diff
@@ -0,0 +1,55 @@
+// From the .src legacy ButtonTableExport
+
+const normalizeString = (string = '') => string.trim().replace(/\s/g, ' ')
```

Comment:
can u check if you can use Strings.normalize

Reply:
From the docs: _"In JavaScript, normalize() is a string method that is used to convert a string to its Unicode Normalization Form."_ In our case we want to normalize in a different way (trimming spaces at both ends of the string and then replacing any type of space to a simple space).